### DISCUSSION TITLE
Ajax admin

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#34551] [Q&A]
Is there a reason that we dont use ajax in the admin for things like list view pagination - saves reloading the page everytime etc

**Comment by alikon** [comment]
one. none and a hundred of thousand 

mostly no code...

**Comment by richard67** [comment]
Maybe the reason is our lack of people who have the knowledge? I definitely don't have it.

**Reply by dgrammatiko** [reply]
> Maybe the reason is our lack of people who have the knowledge? I definitely don't have it.

Actually, this is extremely easy to implement but it requires changes in the template, eg serving the shell (https://developers.google.com/web/fundamentals/architecture/app-shell) differently than the content. Ans just to prove that my opinion that this is extremely easy I can point to https://github.com/dgrammatiko/sloth/blob/step-6-inherritable/sloth/header.php (with a small difference there, the shell is broken to 2 files: header.php + footer.php due to the composition in the service worker, but the idea is the same)
